inspire/modules/deposit/workflows/literature.py

```diff
@@ -327,10 +335,11 @@ def remove_exclusive_fields(fieldlist):
             delete_keys.extend(thesis_fields)
 
         if 'defense_date' in metadata and metadata['defense_date']:
-            if 'note' in metadata and metadata['note']:
-                metadata['note'] += ', presented on ' + metadata['defense_date']
+            defense_note = {'value': 'Presented on ' + metadata['defense_date']}
+            if metadata.get('note', None):
+                metadata['note'].append(defense_note)
```

Comment:
There is a problem here in that it is not backwards compatible with existing submission data. We would probably need to add an ugly check if the value is a string.

Old submission data:
```
In [6]: m.get("note")
Out[6]: u'4 pages, 2 figures'
```
will get an error when trying to `list.append`: 
```
AttributeError: 'unicode' object has no attribute 'append'
```